src/transactionsFactories/tokenManagementTransactionIntentsFactory.spec.ts

```diff
@@ -218,4 +219,158 @@ describe("test token management transactions factory", () => {
+        assert.equal(transaction.sender, grace.address.toString());
+        assert.equal(transaction.receiver, grace.address.toString());
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.gasLimit, 60164000n);
+    });
+
+    it("should create 'Transaction' for modifying creator", () => {
+        const transaction = tokenManagementFactory.createTransactionForModifyingCreator({
+            sender: grace.address,
+            tokenIdentifier: "TEST-123456",
+            tokenNonce: 1n,
+        });
+
+        assert.deepEqual(transaction.data, Buffer.from("ESDTModifyCreator@544553542d313233343536@01"));
+        assert.equal(transaction.sender, grace.address.toString());
+        assert.equal(transaction.receiver, grace.address.toString());
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.gasLimit, 60114500n);
+    });
+
+    it("should create 'Transaction' for updating metadata", () => {
+        const transaction = tokenManagementFactory.createTransactionForUpdatingMetadata({
+            sender: grace.address,
+            tokenIdentifier: "TEST-123456",
+            tokenNonce: 1n,
+            tokenName: "Test",
+            royalties: 1234n,
+            hash: "abba",
+            attributes: Buffer.from("test"),
+            uris: ["firstURI", "secondURI"],
+        });
+
+        assert.deepEqual(
+            transaction.data,
+            Buffer.from(
+                "ESDTMetaDataUpdate@544553542d313233343536@01@54657374@04d2@61626261@74657374@6669727374555249@7365636f6e64555249",
+            ),
+        );
+        assert.equal(transaction.sender, grace.address.toString());
+        assert.equal(transaction.receiver, grace.address.toString());
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.gasLimit, 60218000n);
+    });
+
+    it("should create 'Transaction' for recreating metadata", () => {
+        const transaction = tokenManagementFactory.createTransactionForNftMetadataRecreate({
+            sender: grace.address,
+            tokenIdentifier: "TEST-123456",
+            tokenNonce: 1n,
+            tokenName: "Test",
+            royalties: 1234n,
+            hash: "abba",
+            attributes: Buffer.from("test"),
+            uris: ["firstURI", "secondURI"],
+        });
+
+        assert.deepEqual(
+            transaction.data,
+            Buffer.from(
+                "ESDTMetaDataRecreate@544553542d313233343536@01@54657374@04d2@61626261@74657374@6669727374555249@7365636f6e64555249",
+            ),
+        );
+        assert.equal(transaction.sender, grace.address.toString());
+        assert.equal(transaction.receiver, grace.address.toString());
+        assert.equal(transaction.value, 0n);
+        assert.equal(transaction.gasLimit, 60221000n);
+    });
+
+    it("should create 'Transaction' for changing to dynamic", () => {
+        const transaction = tokenManagementFactory.createTransactionForMakingTokenDynamic({
```

Comment:
Indeed, the name sounds good. However, for the others we've keep the name "mirrored" to the name of the builtin function (more or less). We can also say `createTransactionForChangeToDynamic` or `createTransactionForChangingTokenToDynamic`. Or `createTransactionForChangingToDynamicToken`, which is also consistent with `createTransactionForRegisteringDynamicToken` from below.

Reply:
renamed to `createTransactionForChangingTokenToDynamic`